### DISCUSSION TITLE
I can't add a custom world (iris) to the claim

### DISCUSSION THREAD

**Discussion by patrick323** [GriefPrevention/GriefPrevention#2403] [Q&A]
I'm trying to add my new world generated by the iris plugin, I put the world name and survival mode, but when I restart the server the configuration comes back and adds the custom world that I added, what could it be?

**Comment by Jikoo** [comment]
See #2187 - if you can provide the full name of this plugin (including correct capitalization!) it is relatively easy to add compatibility. Alternately, you can contact them and ask them to add GriefPrevention to their loadBefore.

In an ideal world, all world loading plugins should load before any plugins with the (default) postworld load order, but there's no way to specify a stage after vanilla world loading but before world-dependent plugins.
